modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -1131,28 +1131,36 @@ protected function render_tab_containers_html( $item_settings ): string {
 	 * @param int $index
 	 * @param array $item_settings
 	 */
-	public function print_child( $index, $item_settings = [] ) {
-		$children = $this->get_children();
-		$child_ids = [];
-
-		foreach ( $children as $child ) {
-			$child_ids[] = $child->get_id();
-		}
-
-		// Add data-tab-index attribute to the content area.
-		$add_attribute_to_container = function ( $should_render, $container ) use ( $item_settings, $child_ids ) {
-			if ( in_array( $container->get_id(), $child_ids ) ) {
-				$this->add_attributes_to_container( $container, $item_settings );
-			}
-
-			return $should_render;
-		};
-
-		add_filter( 'elementor/frontend/container/should_render', $add_attribute_to_container, 10, 3 );
-		$children[ $index ]->print_element();
-		remove_filter( 'elementor/frontend/container/should_render', $add_attribute_to_container );
+	public function print_child($index, $item_settings = []) {
```

Comment:
Hi @thorbergurharaldsson 
Thanks for the PR.
Can you please keep the original spacing? The PHP Lint is failing: https://github.com/elementor/elementor/actions/runs/8801041716/job/24155726088?pr=25817
This diff should be 3-5 lines in total.